synse/scheme.go

```diff
@@ -0,0 +1,108 @@
+package synse
+
+// scheme.go defines the scheme for the server's responses.
+
+// Error describes an error response.
+type Error struct {
+	HTTPCode    int    `json:"http_code"`
+	ErrorID     int    `json:"error_id"`
+	Description string `json:"description"`
+	Timestamp   string `json:"timestamp"`
+	Context     string `json:"context"`
+}
+
+// Status describes a response for `/test` endpoint.
```

Comment:
can open an issue - once we have synse v3 actually in place, we can update the documentation to include a link to the spec